### ISSUE TITLE
Conditional parameters are not shown appropriately

### ISSUE THREAD

**Issue by nabenabe0928** [optuna/optuna-dashboard#734]
### Description

In the trial table, parameters are not shown correctly and they will be shown in the json format when the search space has (a) conditional parameter(s).

 
![image](https://github.com/optuna/optuna-dashboard/assets/47781922/fe6b8197-c7cb-4de5-aa63-2af95d57ade2)

### How to Reproduce

Run the following:

```python
import optuna


def objective(trial):
    category = trial.suggest_categorical("category", ["foo", "bar"])
    if category == "foo":
        return (trial.suggest_float("x1", 0, 10) - 2) ** 2
    else:
        return -((trial.suggest_float("x2", -10, 0) + 5) ** 2)


study = optuna.create_study(storage="sqlite:///test.db")
study.optimize(objective, n_trials=100)
```

And then check on the dashboard:

```shell
$ optuna-dashboard sqlite:///test.db
```

### Python version

3.9

### Optuna version

3.5.0

### optuna-dashboard version or git revision

0.14.0

### Web browser

Google Chrome

**Comment by adjeiv** [comment]
https://github.com/optuna/optuna-dashboard/blob/main/optuna_dashboard/ts/components/TrialTable.tsx#L94

Seems to be due to this condition, which is probably evaluating to `false` in the case of the conditional parameters. Did you have in mind to take the union of all set parameters, have a column for each, and for each trial to use the value if it exists? I only wonder whether the initial design was to prevent lots of blank column values.

**Comment by nabenabe0928** [comment]
This issue has been addressed by #740 .
